site/styles.css

```
@@ -1,194 +1,211 @@
-  --accent-green-soft: color-mix(in srgb, var(--wg-green) 12%, transparent);
-  --accent-blue-soft: color-mix(in srgb, var(--wg-blue) 14%, transparent);
-  --accent-yellow-soft: color-mix(in srgb, var(--wg-yellow) 10%, transparent);
-  --accent-red-muted: color-mix(in srgb, var(--wg-red) 62%, var(--label));
-  --accent-green-muted: color-mix(in srgb, var(--wg-green) 58%, var(--label));
-  --accent-blue-muted: color-mix(in srgb, var(--wg-blue) 65%, var(--label));
-  --accent-yellow-muted: color-mix(in srgb, var(--wg-yellow) 55%, var(--label));
+  color-scheme: dark;
 
+  --display: "Space Grotesk", "Segoe UI Variable Display", "Segoe UI", system-ui, sans-serif;
+  --sans: "Segoe UI Variable Text", "Segoe UI", system-ui, -apple-system, sans-serif;
+  --mono: "JetBrains Mono Web", ui-monospace, "Cascadia Mono", Consolas, monospace;
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)